### PR TITLE
Updated the README

### PR DESCRIPTION
**Pull request by danielc-evans**

### CODE REVIEW

**Comment by jayjb** [comment]
Hi @danielc-evans,

Thanks for the more clear rewording. Ill run it past some others and hit you back soon.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,12 @@ Thinkst Applied Research
 Overview
 ----------
 
-OpenCanary is a daemon that runs several canary versions of services that alerts when a service is (ab)used.
+In essence, OpenCanary creates a network honeypot allowing you to catch hackers before they fully compromise your systems. As a technical definition, OpenCanary is a daemon that runs several canary versions of services that alerts when a service is (ab)used. 
+
+Features
+----------
+
+* Recieve email alerts as soon as potential threats are detected, highlighting the threat source IP address and where the breach may have taken place.
```
Comment (jayjb) [inline README.md:15]: spelling correction here: `receive`
I figure if we creating a `Features` section, we should add a few other alerting mechanisms too then. Webhooks, syslog etc. 
We can also add the services that are offered. What do you think?

**Review by jayjb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (danielc-evans) [reply README.md:15]: Sounds good. Feel free to add those sections.

Comment (jayjb) [reply README.md:15]: Cool, let me know once the spelling correction is made

**Review by jayjb — CHANGES_REQUESTED** [review]

**Review by jayjb — APPROVED** [review]

**Comment by jayjb** [comment]
Thanks @danielc-evans for the rewording.

**Pull request merged**